### DISCUSSION TITLE
Feature Request: removing Alt hotkey from showing/hiding menu

### DISCUSSION THREAD

**Discussion by ghost** [wireapp/wire-desktop#5074] [Ideas]
Platform: GNU\Linux (or any other open source nix-like system with Xorg and window manager)
The problem is:
Xorg is using ab Alt-hold hotkey to move a window/frame using any it's point for dragging.
Hotkeys for switching  workspaces (virtual desktops) usually uses Alt + … hotkeys.
So, any time a user is using Alt-hotkey, the status of wire-app changes.
This looks not very good.

**Comment by ffflorian** [comment]
Hi @thisnickwasfree, the Alt key behavior was implemented due to a bug fix reported in https://github.com/wireapp/wire-desktop/issues/901. Unfortunately we can't provide support for all window managers and Linux support is experimental.

**Comment by ghost** [comment]
>we can't provide support for all window managers
There is no need in support ALL window managers, supporting Xorg is enough.
And the issue is not about supporting of something. It's about bad practice of using modification (!) keys as hotkeys.
>Linux support is experimental
Hm. As for me, I do not need messenger without Linux support at all.
